packages/react-percy-test-framework/src/interfaces/getCommonInterface.js

```diff
@@ -25,15 +27,20 @@ export default function getCommonInterface(suites) {
       suites.unshift(suite);
       if (typeof fn === 'function') {
         await fn.call(suite);
-      } else {
-        throw new Error(`Suite "${suite.fullTitle()}" was defined but no callback was supplied.`);
       }
       suites.shift();
       return suite;
     },
     snapshot(title, options, fn) {
       const snapshot = new Snapshot(title, options, fn);
       suites[0].addSnapshot(snapshot);
+      const snapshotFullTitle = snapshot.fullTitle();
+      if (snapshotNames[snapshotFullTitle]) {
+        throw new Error(
+          `A snapshot named \`${snapshotFullTitle}\` has already been added, please use a different name`,
```

Comment:
Yup! Once we have a solid V1 out, adding really helpful error messages and suggestions like that would be awesome. From the testing and build tools I've used, Jest is the gold standard example of awesome error messages, so we should definitely follow their example.